recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ test:
 
 about:
   home: https://iris-grib.readthedocs.io
-  license: LGPL-3.0
+  license: LGPL-3.0-only
```

Comment:
@pp-mo 
Why `only`?
I notice that Iris has `LGPL-3.0-or-later`, should they not be the same?

Reply:
Yup, go with `LGPL-3.0-or-later` :+1: